src/legacy/core_plugins/vis_type_timeseries/index.ts

```diff
@@ -36,18 +36,36 @@ const metricsPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyPlu
       styleSheetPaths: resolve(__dirname, 'public/index.scss'),
       hacks: [resolve(__dirname, 'public/legacy')],
       injectDefaultVars: server => ({}),
+      mappings: {
+        'tsvb-validation-telemetry': {
+          properties: {
+            failedRequests: {
+              type: 'long',
+            },
+          },
+        },
+      },
+      savedObjectSchemas: {
+        'tsvb-validation-telemetry': {
+          isNamespaceAgnostic: true,
+        },
+      },
     },
-    init: (server: Legacy.Server) => {
+    init: async (server: Legacy.Server) => {
       const initializerContext = {} as PluginInitializerContext;
-      const core = { http: { server } } as CoreSetup & CustomCoreSetup;
+      const core = {
```

Comment:
I don't think that we need it any more. Looks like 'http' service already in server.newPlatform.setup.core

![image](https://user-images.githubusercontent.com/20072247/71175706-3db6d180-2279-11ea-9992-e69eed370f5d.png)

Can we refactor it in the following way:

![image](https://user-images.githubusercontent.com/20072247/71176029-eebd6c00-2279-11ea-9b43-8bdafd6f200a.png)

Reply:
Eventually yes, but not yet because the legacy server functions very different than `coreSetup.http`. I plan to do this migration in a separate PR.